### ISSUE TITLE
Searchbox/Textfield: No visual difference when disabled in high contrast mode

### ISSUE THREAD

**Issue by micahgodbolt** [OfficeDev/office-ui-fabric-react#10624]
pulled from #7500

A standard `<input disabled />` is not visually different from an enabled one in high contrast mode (not something we broke in Fabric). Disabled labels look different, but the inputs (borders/background) and place holder text are identical.

![image](https://user-images.githubusercontent.com/1434956/65621535-cbd03400-df78-11e9-9a8b-5a19032b22e5.png)

TextField

![image](https://user-images.githubusercontent.com/1434956/65621614-f15d3d80-df78-11e9-9714-4333686351d2.png)

**Comment by angallison** [comment]
@betrue-final-final said these should be grayText. Do you need anything else for this?

**Comment by betrue-final-final** [comment]
The border should also be GrayText

**Comment by dzearing** [comment]
@ecraig12345 @micahgodbolt Is this resolved? It still says it needs triage.

**Comment by micahgodbolt** [comment]
bot added needs triage after my self assign and tagging. cleaned up
